src/components/timeline/Extensionboth.jsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import { Body2, Heading3 } from '../shared';
+import {
+  Container,
+  BoxWrapper,
+  MobileContainer,
+  LineWrapper,
+  RightLineWrapper,
+  VerticalBox,
+  WheelWrapper,
+  ArrowWrapper,
+  EllipseWrapper,
+  CircledLineWrapper,
+  Mobile,
+  Desktop,
+  Date,
+  Month,
+  EllipseContent,
+} from './styles';
+
+export const Box = ({ node }) => (
+  <BoxWrapper color={node.boxcolor}>
+    <div>
+      <Heading3 semibold>{node.title}</Heading3>
+      <img
+        src='https://res.cloudinary.com/dgjzygzgx/image/upload/v1698431749/Line_3_o1ztvt.svg'
+        alt='line'
+      />
+    </div>
```

Comment:
Why is there an extra div here? Move the links to content folder.

Reply:
The div is the wrapper for H3 and img. have used it for alignment

Reply:
Check if that can be done with boxwrapper

Reply:
Yes that was possible. Done!